provisioner/registryv1/README.md

```diff
@@ -0,0 +1,123 @@
+# registryv1 Provisioner
+
+## Summary
+
+The registryv1 provisioner is a core rukpak provisioner that knows how to interact with `registry+v1` format bundles. 
+The `registry+v1` bundles are the typical OLM bundles that contain a CSV manifest as well as other supported manifest types, such as CRDs and RBAC.
+They contain a manifests and metadata directory and can be built via standard OLM tooling such as the `opm` tool.
+For more information on the `registry+v1` bundle format see the [documentation](https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/#operator-bundle). 
+
+The registryv1 provisioner is able to unpack a given `registry+v1` bundle onto a cluster and then instantiate it, making the
+content of the bundle available in the cluster. It does so by reconciling the `Bundle` and `BundleInstance` types that 
+have the spec.provisionerClassName field set to `core.rukpak.io/registry+v1`. This field must be set to the correct provisioner name 
```

Comment:
the `spec.provisionerClassName` field should be documented on the main README as well, even if we remove references to the `ProvisionerClass` API from there.